schemas/value_sources.json

```diff
@@ -106,5 +106,22 @@
         "$ref": "https://eq.ons.gov.uk/value_sources.json#/location_value_source"
       }
     ]
+  },
+  "any_value_source_except_location": {
+    "description": "Any value source",
+    "anyOf": [
+      {
+        "$ref": "https://eq.ons.gov.uk/value_sources.json#/answer_value_source"
```

Comment:
`answer_value_source` has `list_item_selector` property which in turn has `location_value_source` ?

Reply:
That's fine, as that will still resolve to an answer value.